### PR TITLE
Fix case handling of view column names

### LINKED ISSUE #4232: View is stale immediately after creation
Presto version is  presto-server-0.123  user
My config ：
connector.name=hive-cdh5
hive.metastore.uri=thrift://HADOOP1:9083
1. I  operated CREATE VIEW  view_nginx_log AS 
2. CREATE OR REPLACE VIEW  view_nginx_log AS 
   SELECT   ip,...  USER,
   TIME,...line_no
   FROM
   (SELECT  ip, ..  USER,
     TIME,..url_extract_parameter(url,'lineNo') AS line_no
   FROM
     txt_nginx 
   WHERE DATE = '20151224' 
     AND HOUR = '15' 
   UNION
   ALL 
   SELECT  ip, .....url_extract_parameter(url,'lineNo') AS line_no  
   FROM
   orc_nginx_log);
3. select \* from view_nginx_log
   show me a error: 【Query 20151224_093318_00023_wvykb failed: View 'hive.default.view_nginx_log' is stale; it must be re-created】

I  drop the view and re-created ，see the same error。
i think 
【USER, TIME】  sources of errors

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
Fixes #4232. I hope `AbstractTestIntegrationSmokeTest` is a good place to add a test for this.

### CODE REVIEW

**Comment by nezihyigitbasi** [comment]
Added an analyzer unit test.

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java [hunk 1]
```diff
@@ -228,6 +228,16 @@ public void testNonQueryAccessControl()
     }
 
     @Test
+    public void testViewCaseSensitivity()
```
Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java]: This is testing engine functionality, so it should go in `AbstractTestDistributedQueries`. This class is for a limited set of tests that can quickly test the basic functionality of a connector (and thus is fast enough to run for every connector as part of the normal build).

Some existing tests in this class like `testViewAccessControl` are actually in the wrong place and should be moved out.

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java [hunk 2]
```diff
@@ -228,6 +228,16 @@ public void testNonQueryAccessControl()
...
+            throws Exception
+    {
+        computeActual("create view test_view_uppercase as select X from (select 123 X)");
```
Comment (electrum) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java]: Nit: capitalize SQL keywords like `CREATE VIEW` in tests

File: presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java (hunk 1 above)
Comment (nezihyigitbasi) [inline presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java]: gotcha, moved the tests.

**Comment by martint** [comment]
Merged, thanks!

**Pull request closed without merging**